### DISCUSSION TITLE
[Fall 2024] Week 1 Assignment

### DISCUSSION THREAD

**Discussion by jooohyunpark** [jooohyunpark/itp-canvas-for-coders#3] [Assignments]
1. Read [three.js fundamentals](https://threejs.org/manual/#en/fundamentals).
2. Start your (Vite) project.
3. Explore projects made with three.js. Be prepared to present your favorite. Explain why it impresses you and what aesthetics or narratives make it captivating.

**Comment by ipwu** [comment]
https://www.awwwards.com/sites/who-is-playing-the-guqin
[ https://pvp.qq.com/act/6749/a20210927pf/index.html ]

I am impressed with the watercolor bleed effect used with the mouse clicks, and the interaction with the guqin notes tied together with the clicks or slide of the mouse.

**Comment by Joyceyyzheng** [comment]
http://goinghome.302chanwoo.com/
(This artist has other projects too which I also like)
Awesome interaction, animation and scene transitions. It's a great balance of using three.js to curate a story and involves small but delicate user interaction to let users immerse in the story.

**Comment by hds415** [comment]
http://falter.wild.plus/
I think the illustration, music, and interactive elements work very well together to create a particular aesthetic and also to deliver a story. Simple ways to interact but felt very immersive.

I also really liked the going home project, but figured someone already mentioned it above so decided to skip! I'll drop a link for reference. http://goinghome.302chanwoo.com/

**Comment by ZoeyFerguson0** [comment]
https://madebyevan.com/webgl-water/

Water has always been calming to me in a way that feels opposite to technology; it's natural, fluid, and completely out of my control. This example of three.js puts the water in your control but allows you to witness, affect, and pause the beautiful ripples and reflections of water in ways I've always wished to do. It is a simple example yet seems crazy difficult to replicate!

**Comment by wallitah** [comment]
https://pavilion-thefutureinmind.akufen.ca/cube/the-jewel

This site is part of a larger, multi-layered project. I really enjoyed exploring this section because the art style, diverse interactions, and music came together to create a soothing narrative.

The choice to center the 'interaction window' as a frame was interesting, though I wonder if making it full screen might have been more impactful. I'm not entirely sure how it all ties into Canada (lol), but I appreciated seeing a different art style from the usual 3D / WebGL aesthetic that I'm sure to seeing in the browser.

**Comment by lqiuuu** [comment]
https://tympanus.net/Development/3DTypeEffects/index.html
Just found this cool website and wanted to share! This creates effects on the texts you type in.
